create-service-and-ensure-existence.sh

```diff
@@ -0,0 +1,18 @@
+organization=$1
+application=$2
+service=$3
+
+curl -X POST "$API_ENDPOINT" \
+-u $USER_LOGIN:$USER_PASSWORD \
```

Comment:
These need to prefixed with MSAT_ - I don't think they are 'login' and 'password' - api access....

@andrevinsky